### DISCUSSION TITLE
Frigate+ Free vs Plus account

### DISCUSSION THREAD

**Discussion by tlundste** [blakeblackshear/frigate#4704] [Ask A Question] [answered]
What's the difference between Frigate+ Free, and Frigate+ Plus accounts?  What does Free get you (other than the ability to contribute)?  I get that the Plus account gets you access to custom models.  But I'm confused as to the benefit of the Free account.
Is there some sticky-post somewhere I can't easily see that explains it all?  The "Frigate integration" listed on the Friage+ page isn't clear to me exactly what that is.

**Comment by NickM-27** [comment]
Currently the custom models are still not available, so frigate+ is the only option. As a free user you can contribute your examples. The "Frigate+ integration with frigate" means how you can submit examples from Frigate directly. This integration will also be improved in future versions to give more abilities inside Frigate itself.

As far as how model access and other things develop in the future, I don't believe that has been specified yet (and is not up to me)

**Comment by tlundste** [comment]
OK, I'll see if the author will reply, maybe he'll have more insight and indicate if the future of the Free version includes anything more than just the ability to contribute images.
Thx!

**Comment by blakeblackshear** [comment, marked as answer]
Frigate+ free accounts just offer the ability to upload examples to be used to create optimized models for your cameras. You don't get anything other than that with a free account, which is why it's free.

**Reply by HacktonGwill** [reply]
Hi, how are you, just want to be more clear what it means for : " be used to create optimized models ", is that models automatically imported into the local frigate or does that mean it just form a global training pool and if needed you will pick up them and create a universal model? thanks so much in advance

**Reply by blakeblackshear** [reply]
I use the uploaded images to create a global training pool and models tuned to your cameras. These will be available for use to subscribers.

**Reply by HacktonGwill** [reply]
awesome, thank you! will be surely subscribe the plus account as soon as it is avaliable

**Reply by User873902** [reply]
@HacktonGwill thanks for seeking clarification. If I understand correctly, there's no user benefit to free-contribute now if we do not plan on subscribing to Frigate+ the future, other than perhaps the general 'thank you' to the developers for the hard work. Is it even helpful for Frigate to receive uploads if we don't need models specifically tuned to our cameras?

If it is beneficial to the project, I wonder if it would be beneficial to create an upload incentivisation to better amass the labeled training data. Perhaps one free month of Frigate+ for every five uploads. Low enough that people wouldn't upload bad examples, but high enough that the return for the project is worth it.

**Reply by disconn3ct** [reply]
How do we withdraw our images from the training data?

**Reply by NickM-27** [reply]
You should email blake@frigate.video

**Reply by gigabyte-1000** [reply]
Is this still true? Or is deleting the image from the website now enough to remove it from the training data?
I thought free-users would get access to a basic model just not custom models, so after uploading some images I realized that's not the case.
